woocommerce/woocommerce-android
# AI product description: Promo dialog

Fixes #9255, a subtask of #9214.

This PR adds a new promo dialog for the AI product description to the My store screen. The dialog buttons don't do anything. It is currently hidden behind a feature flag.

![image](https://github.com/woocommerce/woocommerce-android/assets/1522856/30b74927-1dcc-44b6-914a-663891b7af52)

**To test:**

1. Start the app in debug mode
2. Log in, if logged out
3. Notice the AI product description dialog is shown when the My store screen is loaded
